### PR TITLE
Init_Project

### PR DESCRIPTION
**Pull request by ngyihkai**
Ready

### CODE REVIEW

**Comment by ngyihkai** [comment]
Ready

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (ghost) [inline config/database.yml]: 1. Config this file into .gitignore then run bellow command line --> do not push this file to pull request
git rm --cached config/database.yml
2. Create a new file with name "config/database.yml.example" --> this file using push to pull request

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: Research and apply "i18n" into project for translate string (using display on UI for end user)
https://guides.rubyonrails.org/i18n.html#providing-translations-for-internationalized-strings

File: config/credentials.yml.enc [hunk 3]
```diff
@@ -0,0 +1 @@
+aMclSLfxhNA0UmyhyeU+x96T6j0aKeI3bPanMs4RurhiyATSph/JQxL5gZwU6ZekDo6Jb4i4jC5iMTlNNe/e/Pij9LVZ2DvJkKtkAbPEzy6Y+wIRMO4aVCzWqNsd4T6ZDNerkf0OugHicj4yadsjPBUHJT9UeV9tju2rswY6UDkx5L1IPoMxJzPNZSSpxP/ATLJyc9tGQr/3bW1mrVIix8zXRBzAbld96SUlQiSkVpLCcGN96IfkYvhsol1HTLtzq4yABL4RUCO8e0VPtr04v4WJSU+BSIljNWtwmcMjbVzmweTo8Cx2YBNnrle3H/haR2I9OQUe/fUtm9awLuCcBmN0WB9y3WC+Z/M0MwHBG++IWeLNOs6emibhpvB55Fh2z4mfICsT+FalvdkA/ammmoYW0QNxYmYX9u+t--7LRf03Skibyy7fv1--6ExwLt521KnLHP9SZO/Zww==
```
Comment (ghost) [inline config/credentials.yml.enc]: gitignore this file

**Review by ghost — COMMENTED** [review]
cmt

File: config/credentials.yml.enc (hunk 3 above)
Comment (ngyihkai) [reply config/credentials.yml.enc]: is it copy this to my buttom part of my gitignore file?

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (ngyihkai) [reply app/views/layouts/application.html.erb]: is it i need use YAML (.yml) or plain Ruby (.rb) files for storing my translations in SimpleStore?

File: config/credentials.yml.enc (hunk 3 above)
Comment (ngyihkai) [reply config/credentials.yml.enc]: after adding this file, i need to push again right?

Comment (ngyihkai) [reply config/credentials.yml.enc]: after i push, terminal wrote everything is up-to-date. How can i update again in github?

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (ghost) [reply app/views/layouts/application.html.erb]: You should use .yml

File: config/credentials.yml.enc (hunk 3 above)
Comment (ghost) [reply config/credentials.yml.enc]: after adding file you have to commit it 
try to follow  this

git add .
git comit --amend --no-edit
git push origin init_project -f

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (ngyihkai) [reply app/views/layouts/application.html.erb]: In my application.html.erb should be like this?
en:
  hello_world: Hello world!
  hello_flash: Hello flash
  success:
    'true':  'True!'
    'on':    'On!'
    'false': 'False!'
  failure:
    true:    'True!'
    off:     'Off!'
    false:   'False!'

pirate:
  hello_world: Ahoy World
  hello_flash: Ahoy Flash

I18n.t 'success.true'  # => 'True!'
I18n.t 'success.on'    # => 'On!'
I18n.t 'success.false' # => 'False!'
I18n.t 'failure.false' # => Translation Missing
I18n.t 'failure.off'   # => Translation Missing
I18n.t 'failure.true'  # => Translation Missing

File: config/credentials.yml.enc (hunk 3 above)
Comment (ghost) [reply config/credentials.yml.enc]: if this is a copy file, you don't need to  gitignore it

**Comment by ghost** [comment]
cmt

File: config/credentials.yml.enc (hunk 3 above)
Comment (ngyihkai) [reply config/credentials.yml.enc]: i just push it already. Can u commit which part I still need to correct?

**Comment by ngyihkai** [comment]
Ready

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (ngyihkai) [reply app/views/layouts/application.html.erb]: sorry, just found that i copy those coding to the wrong files

File: config/database.yml (hunk 1 above)
Comment (ngyihkai) [reply config/database.yml]: does it mean that copy all the coding from the database.yml to the .gitignore?
after this run that code?

**Comment by ngyihkai** [comment]
Ready

**Comment by ngyihkai** [comment]
Ready

**Comment by ngocvt-0484** [comment]
merged
i18n can apply on next pull request

**Pull request merged**